drivers/source/usb/mbed_lib.json

```diff
@@ -1,3 +1,3 @@
 {
-    "name": "usb"
+    "name": "drivers-usb"
```

Comment:
What is the impact of this name change?

Reply:
None. The name needs to be unique. This gives it a namespace.

Reply:
I can't see it referenced in any "requires" directive so it should be fine